From 6162c5fd56a1febe1273beef38a3017c83f3c163 Mon Sep 17 00:00:00 2001
From: Remi Collet <fedora@famillecollet.com>
Date: Tue, 9 Aug 2016 11:26:14 +0200
Subject: [PATCH] CDRIVER-1457 add missing in public API

- mongoc_write_concern_is_acknowledged;
- mongoc_write_concern_is_valid;
- mongoc_write_concern_journal_is_set;

Also related to https://jira.mongodb.org/browse/PHPC-762
---
 build/autotools/versions.ldscript | 3 +++
 src/libmongoc.symbols             | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/build/autotools/versions.ldscript b/build/autotools/versions.ldscript
index 68da7c5..d3a68c1 100644
--- a/build/autotools/versions.ldscript
+++ b/build/autotools/versions.ldscript
@@ -368,4 +368,7 @@ LIBMONGOC_1.4 {
         mongoc_uri_set_username;
         mongoc_uri_set_write_concern;
         mongoc_write_concern_append;
+        mongoc_write_concern_is_acknowledged;
+        mongoc_write_concern_is_valid;
+        mongoc_write_concern_journal_is_set;
 } LIBMONGOC_1.3;
diff --git a/src/libmongoc.symbols b/src/libmongoc.symbols
index f7a5510..f555b1f 100644
--- a/src/libmongoc.symbols
+++ b/src/libmongoc.symbols
@@ -345,6 +345,8 @@ mongoc_write_concern_get_w
 mongoc_write_concern_get_wmajority
 mongoc_write_concern_get_wtag
 mongoc_write_concern_get_wtimeout
+mongoc_write_concern_is_acknowledged
+mongoc_write_concern_is_valid
 mongoc_write_concern_new
 mongoc_write_concern_set_fsync
 mongoc_write_concern_set_journal
@@ -352,3 +354,4 @@ mongoc_write_concern_set_w
 mongoc_write_concern_set_wmajority
 mongoc_write_concern_set_wtag
 mongoc_write_concern_set_wtimeout
+mongoc_write_concern_journal_is_set
-- 
2.5.5

